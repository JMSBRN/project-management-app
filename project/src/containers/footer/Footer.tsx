import React from 'react';
import { AutorLink, Autors, FooterWrapper, LogoSchool } from './Footer.style';

const Footer = () => {
  return (
    <FooterWrapper>
      <LogoSchool href="https://rs.school/react/" />
      <Autors>
        <AutorLink href="https://github.com/alexshishkov">Alexey Shishkov</AutorLink>
        <AutorLink href="https://github.com/JMSBRN">Aleksandr Zakhavai</AutorLink>
        <AutorLink href="https://github.com/botino-k">botino-k</AutorLink>
      </Autors>
      Â© 2022. React 2022Q3
    </FooterWrapper>
  );
};

export default Footer;
