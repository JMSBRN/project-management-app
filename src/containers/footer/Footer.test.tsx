import { render } from '@testing-library/react';
import React from 'react';
import Footer from './Footer';

describe('<Footer />', () => {
  it('should be rendering correctly ', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment).toMatchSnapshot();
  });
});
