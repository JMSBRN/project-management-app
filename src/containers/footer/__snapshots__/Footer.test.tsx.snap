// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should be  1`] = `[Function]`;
